README.md

```diff
@@ -173,7 +182,7 @@ application.init().then(() => console.log(application.customFields));
 
 The methods available for each of the above scope types matches the general
 access level defined for each type of
-[API Key](https://developers.evrythng.com/docs/api-scope-and-key-permissions).
+API Key for [apiVersion:2](https://developers.evrythng.com/docs/api-keys-and-key-permissions-v2-1) or [apiVersion:1](https://developers.evrythng.com/docs/api-scope-and-key-permissions).
```

Comment:
Fixed this slug to be:
`https://developers.evrythng.com/docs/api-keys-and-key-permissions-v2`

Reply:
fixed, thanks